README.md

```diff
@@ -5,7 +5,94 @@
 [![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?labelColor=000000&logo=docker)](https://www.docker.com/)
 [![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg?labelColor=000000)](https://sylabs.io/docs/)
 
-## This project is a successor to the [C-WAP pipeline](https://github.com/CFSAN-Biostatistics/C-WAP) and is under development as we prepare for a vs 1.0 release which will be available shortly.  
+## This project is a successor to the [C-WAP pipeline](https://github.com/CFSAN-Biostatistics/C-WAP) and is intended to process SARS-CoV-2 wastewater samples to determine relative variant abundance.  
 
-## Please exercise caution when using code shared on this repo until the version 1.0 release. The results generated by this pipeline are not CLIA certified and should not be considered diagnostic.
+## It has been validated for short reads and longreads. However, a word of caution: Ion-torrent data is being vetted and will be fully available in the next realease. The results generated by this pipeline are not CLIA certified and should not be considered diagnostic.
```

Comment:
Not a fan of saying "validated" here since that implies a specific kind of testing that I don't think we've truly done.  Might be better to say "thoroughly tested" or similar language.  Same comment for "Ion-torrent data is being vetted"... maybe change that to say "Ion-torrent results are being tested"